examples/plugin_extract/plugin.cpp

```diff
@@ -46,9 +46,18 @@ class my_plugin
     }
 
     // (optional)
-    void destroy() {}
+    bool set_config(falcosecurity::set_config_input& i) { return false; }
```

Comment:
I suggest returning true and logging at least in one example plugin, just to check that everything works.